lib/target/users.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Target.Users do
+  @moduledoc """
+  The User context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias Target.Repo
+  alias Target.Users.User
+
+  def list_users do
```

Comment:
Where is this needed?

Reply:
It isn't right now, but it came with the example and I thought it could be useful in the future 👀 but let us not rush into da future